report_pre_thesis/src/sections/experiments/ica_aggregated.tex

```diff
@@ -0,0 +1,34 @@
+\subsection{Experiment: ICA Aggregated Datasets}\label{sec:experiment_ica_aggregated_datasets}
+In Section~\ref{sec:ica_data_preprocessing}, we described how we only use one of the five location datasets for each sample for the ICA training process.
+In this experiment, we use all five location datasets for each sample, and aggregate the results by taking the mean of the shots over the datasets.
```

Comment:
```suggestion
In this experiment, we used all five location datasets for each sample, and aggregated the results by taking the mean of the shots over the datasets.
```
In case you agree it should be past tense. If not, disregard this.